docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
+``your_project/your_project/urls.py``::
+
+    # Registration Views
+    # Eventually these will be auto-registered with the Orchestra URLs, but for
+    # now we need to add them separately.
+    url(r'^orchestra/accounts/',
+        include('registration.backends.default.urls')),
+
+    # Logout then login is not available as a standard django
+    # registration route.
+    url(r'^orchestra/accounts/logout_then_login/$',
+        auth_views.logout_then_login,
+        name='logout_then_login'),
+
+    # Orchestra URLs
+    url(r'^orchestra/',
+        include('orchestra.urls', namespace='orchestra')),
+
+    # Beanstalk Dispatch URLs
+    url(r'^beanstalk_dispatch/',
+        include('beanstalk_dispatch.urls')),
+
+Finally, you'll need to get the database set up. Create your database
+with ``python manage.py migrate``. You'll also want to make sure you have an
+initial worker account set up to try out example workflows. We've provided one
+(username: ``demo``, password: ``demo``) as a fixture which you can load with
+``python manage.py loaddata demo_worker``.
+
+Now Orchestra should be ready to go! If you're confused about any of the above,
+check out our barebones `example project <https://github.com/unlimitedlabs/orchestra/tree/v0.1.0/example_project>`_.
+
+*************
+Run Orchestra
+*************
+
+Now that Orchestra is configured, all that remains is to fire it up! Run your
+django project with ``python manage.py runserver`` (you'll want to switch to
+something more robust in production, of course), and navigate to
+``http://127.0.0.1:8000/orchestra/app`` in your favorite browser.
+
+If you see the Orchestra sign-in page, your setup is working! Logging in as
+the demo user we set up above should show you a dashboard with no available
+tasks.
+
+.. _demo-section:
+
+****************************
+Run the Example Project Demo
+****************************
+
+To give you a feel for what it means to run an Orchestra workflow from end to
+end, we've included a very simple example workflow with two steps, one
+automated and one human. The automated step takes a URL and extracts a random
+image from the page. The human step asks an expert to rate how "awesome" the
+image is on a scale from one to five. If you're interested in how we defined
+the workflow, take a look at `the code <https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/simple_workflow/workflow.py>`_,
+though we walk through a more interesting example in
+:doc:`this documentation <example_use>`.
+
+We've written an interactive script to walk through this simple workflow. To
+run it:
+
+* Make sure you added ``simple_workflow`` to your ``INSTALLED_APPS`` setting
+  following the previous section.
+
+* Pull down the script into your project's root directory (``your_project``, next to
+  ``manage.py``)::
+
+      wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/simple_workflow/run.py
+
+* Run the script::
+
+      python run.py
```

Comment:
my initial high level feedback was to say that these instructions contain a lot of set up (that's alright!), but that it might be better for them to include more context around `run.py` and what happens.  It seems like the context is embedded in the interactivity of `run.py` which is interesting.  Maybe give people a sense that there's a TON of stuff that will hapepn in the console when they run `run.py` and they should expect to learn a lot that way?

also: think about if there's a way to capture the learning here somehow...maybe show a sample run of it?